src/main/scala/com/amazon/deequ/analyzers/Analyzer.scala

```diff
@@ -321,6 +323,18 @@ object Preconditions {
     }
   }
 
+  def isNotNested(column: String): StructType => Unit = { schema =>
+    if (schema.fieldNames.contains(column)) {
```

Comment:
We check if the given column exists, but we ignore if it's missing. 

I think there are 2 options here:
- we assume that the existence of the column had been already checked  before this precondition evaluated (like in case of `isNumeric` and `isString`) -> it's a bit unsafe
- in every column predicate (`isNumeric`, `isString` and `isNotNested`) we implicitly call `hasColumn` precondition and make sure the column exists before we do any further verification

What's your view on this?

Reply:
We implemented our analyzers to always issue a `hasColumn` check first, so this problem should be taken care of.